### PR TITLE
Movie support with exoplayer

### PR DESCRIPTION
**Pull request by hotchemi**
### Overview

- Add support for ExoPlayer with DASH url. Why? We wanna tackle new stuff!
- Add two vector icons to show slide and movie existence.
- Add player activity which shows each sessions video.

### Bugs/Known issues

- As you know I copied PlayerActivity and Renderer code from official ExoPlayer sample. We need more refactor to keep it clean and shows more simple example of Exoplayer.
- We need more feature to use ExoPlayer each features.
- Currently I only added wasabeef url to test. If it is okay I'm gonna add more url.

### Screenshot

memo

### CODE REVIEW

**Comment by konifar** [comment]
Fantastic 🌟

File: app/src/main/java/io/github/droidkaigi/confsched/activity/VideoPlayerActivity.java [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+import java.util.Locale;
+
+import io.github.droidkaigi.confsched.R;
+import io.github.droidkaigi.confsched.player.DashRendererBuilder;
+import io.github.droidkaigi.confsched.player.DemoPlayer;
+
+/**
+ * An activity that plays media using {@link DemoPlayer}.
+ */
+public class VideoPlayerActivity extends Activity implements SurfaceHolder.Callback, OnClickListener,
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/VideoPlayerActivity.java]: Sorry this file is almost copied from exoplayer sample, I'm gonna clean up code in another pr, hopefully... 😇

File: app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java [hunk 2]
```diff
@@ -21,4 +22,10 @@ public static void share(Context context, @NonNull String url) {
         context.startActivity(Intent.createChooser(intent, context.getString(R.string.share)));
     }
 
+    public static void toBrowser(Context context, @NonNull String url) {
```
Comment (hotchemi) [inline app/src/main/java/io/github/droidkaigi/confsched/util/IntentUtil.java]: A tiny refactor.

File: app/src/main/res/raw/sessions_en.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
     "language_id": "ja",
     "slide_url": "https://speakerdeck.com/player/b2127cb508e94238b114512447997aeb",
     "movie_url": "https://www.youtube.com/watch?v=kArtZzDz1xU",
+    "movie_dash_url": "https://manifest.googlevideo.com/api/manifest/dash/initcwndbps/2632500/id/o-AE9RZgFZZANkR7IkphApmRboitWzzC4-tsg9NhuCOb5K/pl/17/source/youtube/mv/m/key/yt6/signature/96E2E5BF1EF5F2E36FE4BF385FBA675121D8A26D.D6B0F02FC33138432A63AE0B0D1AB39DE8D7986B/mm/31/ip/112.138.237.132/mn/sn-8pjuv35p-ogul/ms/au/sver/3/sparams/as%2Chfr%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Cmm%2Cmn%2Cms%2Cmv%2Cpcm2cms%2Cpl%2Cplayback_host%2Crequiressl%2Csource%2Cexpire/itag/0/ipbits/0/as/fmp4_audio_clear%2Cwebm_audio_clear%2Cwebm2_audio_clear%2Cfmp4_sd_hd_clear%2Cwebm2_sd_hd_clear/expire/1468186731/pcm2cms/yes/requiressl/yes/hfr/1/mt/1468164822/upn/dRZOVDIaTsg/playback_host/r3---sn-8pjuv35p-ogul.googlevideo.com/fexp/3300115%2C3300130%2C3300164%2C3313267%2C9406989%2C9416126%2C9416891%2C9422596%2C9428398%2C9431012%2C9433096%2C9433380%2C9433946%2C9435526%2C9435876%2C9437066%2C9437552%2C9439652",
```
Comment (hotchemi) [inline app/src/main/res/raw/sessions_en.json]: This is a trick of getting DASH url :trollface:

**Comment by hotchemi** [comment]
@konifar Here we go! Minimal implementation is ready. After this pr is merged I'm gonna add more and more improvement 👍

File: app/src/main/res/values-ko/strings.xml [hunk 4]
```diff
@@ -115,4 +115,22 @@
     <string name="session_feedback_submit_success">전송했습니다.</string>
     <string name="session_feedback_submit_link">의견을 보내</string>
 
+    <!-- VIDEO FIXME: translation please! -->
```
Comment (hotchemi) [inline app/src/main/res/values-ko/strings.xml]: Translation is needed...

File: app/src/main/res/values-ja/strings.xml [hunk 5]
```diff
@@ -115,4 +115,22 @@
     <string name="session_feedback_submit_success">送信しました。</string>
     <string name="session_feedback_submit_link">フィードバックを送信</string>
 
+    <!-- VIDEO FIXME: translation please! -->
```
Comment (hotchemi) [inline app/src/main/res/values-ja/strings.xml]: Translation is needed...

File: app/src/main/res/values-ar/strings.xml [hunk 6]
```diff
@@ -124,4 +124,22 @@
     <string name="session_feedback_submit_success">إرسال نجحت</string>
     <string name="session_feedback_submit_link">إرسال ملاحظات</string>
 
+    <!-- Video FIXME: translation please! -->
```
Comment (hotchemi) [inline app/src/main/res/values-ar/strings.xml]: Translation is needed...

**Comment by konifar** [comment]
So cool!!!
Can I review this?

**Comment by hotchemi** [comment]
@konifar Sure thing! I couldn't fix all bad points in this PR so I'm gonna improve more and more later... :trollface:

**Comment by konifar** [comment]
Okay! Looks good feature 👍 I'll check it at today's lunch time.

File: app/src/main/res/raw/sessions_en.json (hunk 3 above)
Comment (konifar) [inline app/src/main/res/raw/sessions_en.json]: Just question.
How can I get this DASH url? Because I want to help to fill another session movie dash url.

Comment (hotchemi) [inline app/src/main/res/raw/sessions_en.json]: The following is the trick... :trollface: You can get information from http://www.youtube.com/get_video_info?&video_id=ID and decode it and finally get the DASH url 😅 

http://stackoverflow.com/a/31709557/1288107

**Comment by konifar** [comment]
LGTM! Let's improve this good feature together 💪

**Comment by hotchemi** [comment]
Thx!!!!!!!!!!!!!

**Pull request merged**